Add repos for EL 9 in updating guide (#3378)
theforeman/foreman-documentation: guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc

[id="Updating-Disconnected-{project-context}-on-EL8_{context}"]
= Updating a disconnected {ProjectServer} on {EL} 8

> .Procedure on the connected {ProjectServer}

Reviewer: I suggest to add a short introduction to the procedure about the general workflow. the heading and subheading to short succession are slightly confusing to me.

enabled=1
sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
sslverify = 1

[{RepoRHEL8ServerSatelliteServerProjectVersion}]
name={ProjectName} {ProjectVersion} for RHEL 8 RPMs x86_64
baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/layered/rhel8/x86_64/satellite/{ProjectVersion}/os
enabled=1
sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
sslverify = 1

[{RepoRHEL8ServerSatelliteMaintenanceProjectVersion}]
name={ProjectName} Maintenance {ProjectVersion} for RHEL 8 RPMs x86_64
baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/layered/rhel8/x86_64/sat-maintenance/{ProjectVersion}/os
enabled=1
sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
sslverify = 1
----

> +

Reviewer: ```suggestion
```

[id="Updating-Disconnected-{project-context}-on-EL8_{context}"]
= Updating a disconnected {ProjectServer} on {EL} 8

> You can update your disconnected {Project} on {RHEL} 8 by synchronizing the required repositories on the c
> onnected {Project} and syncing the content to the disconnected {Project} using `reposync`.

Reviewer 2: ```suggestion
You can update your disconnected {Project} on {RHEL} 8 by synchronizing the required repositories on the connected {Project} and syncing the content to the disconnected {Project} using `reposync`.
```

.Procedure on the connected {ProjectServer}
. Ensure that you have synchronized the following repositories in your connected {ProjectServer}:
+
* {RepoRHEL8BaseOS}
* {RepoRHEL8AppStream}
* {RepoRHEL8ServerSatelliteServerProjectVersion}
* {RepoRHEL8ServerSatelliteMaintenanceProjectVersion}
. Download the debug certificate of the organization and store it locally at `/etc/pki/katello/certs/org-debug-cert.pem` or a location of your choosing.
For more information, see {AdministeringDocURL}Creating_an_Organization_Debug_Certificate_admin[Creating an Organization Debug Certificate] in _{AdministeringDocTitle}_.
. Create a Yum configuration file under `/etc/yum.repos.d`, such as `_{project-context}-disconnected_.repo`, with the following contents:
+
[options="nowrap" subs="attributes"]
----
[{RepoRHEL8BaseOS}]
name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)

> baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/rhel8/8/x86_64/baseos/os

Reviewer 3: I know this is copied from the old proc, but `_` for italics doesn't work if you don't do `subs="quotes"`. In other places we set `subs="+quotes,verbatim,attributes"` so probably should standardize on everywhere?

On first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
+
[options="nowrap" subs="+quotes,verbatim,attributes"]
----
# {foreman-maintain} update check \
--whitelist="check-upstream-repository,repositories-validate"
----
. Review the results and address any highlighted error conditions before performing the update.
. Due to the lengthy update time, use a utility such as `tmux` to suspend and reattach a communication session.
You can then check the update progress without staying connected to the command shell continuously.
+
If you lose connection to the command shell where the update command is running, you can see the logged messages in the `{installer-log-file}` file to check if the process completed successfully.
. Perform the update:
+
[options="nowrap" subs="+quotes,verbatim,attributes"]
----
# {foreman-maintain} update run \
--whitelist="check-upstream-repository,repositories-setup,repositories-validate"
----

include::snip_steps-needs-reboot.adoc[]

.Additional resources

> * To restore the backup of the {ProjectServer} or {SmartProxyServer}, see {AdministeringDocURL}Restoring_from_a_Full_Backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a Backup]

Reviewer 2: ```suggestion
* To restore the backup of the {ProjectServer} or {SmartProxyServer}, see {AdministeringDocURL}Restoring_from_a_Full_Backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a Backup].
```
